Doc/library/functions.rst

```diff
@@ -687,7 +687,7 @@ are always available.  They are listed here in alphabetical order.
 
    Convert a *value* to a "formatted" representation, as controlled by
    *format_spec*.  The interpretation of *format_spec* will depend on the type
-   of the *value* argument, however there is a standard formatting syntax that
+   of the *value* argument, however, there is a standard formatting syntax that
```

Comment:
Similarly, if you want to use a comma here, you need to change the one preceding "however" with a semi-colon.

Reply:
It would create a different meaning to the sentence. Please read from "The Interpretation.....